tests/features/solution/solution.edit.feature

```diff
@@ -0,0 +1,69 @@
+@api
+Feature: Solution editing.
+  In order to manage solutions
+  As a moderator or solution facilitator
```

Comment:
Remove the moderator! we are only testing as a facilitator. 
We are using the strict model of Og. It means that users with site-wide rights do not have them applied in groups. 
Moderators do not currently have this kind of access.